lib/matplotlib/backend_tools.py

```diff
@@ -1018,6 +1020,68 @@ def _mouse_move(self, event):
         self.toolmanager.canvas.draw_idle()
 
 
+class HelpTool(ToolToggleBase):
+    description = 'Print tool list, shortcuts and description'
+    default_keymap = rcParams['keymap.help']
+    image = 'help.png'
+
+    def __init__(self, *args):
+        ToolToggleBase.__init__(self, *args)
+        self.text_axes = None
+
+    def enable(self, *args):
+        self.text_axes = self.figure.add_axes((0, 0, 1, 1))
```

Comment:
I think you should create the axes with a custom (internal) label (with a comment) to workaround https://github.com/matplotlib/matplotlib/issues/9024 (although that bad behavior is now deprecated), just in case.